### PR TITLE
[17.0][OU-ADD] digest: migration to 17.0

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
/ocabot migration digest

<dependency>Depends on : 

- portal : #4467
- resource : #4471</dependency>

File: openupgrade_scripts/scripts/digest/17.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env, "digest", "17.0.1.1/noupdate_changes.xml")
+    openupgrade.set_xml_ids_noupdate_value(
+        env,
+        "digest",
+        [
+            "digest_tip_digest_1",
```
Comment (remi-filament) [inline openupgrade_scripts/scripts/digest/17.0.1.1/pre-migration.py:13]:
```suggestion
            "digest_tip_digest_0",
            "digest_tip_digest_1",
```
I think you are missing the first one no ?

**Review by remi-filament — CHANGES_REQUESTED** [review]
I think you missed one of the record to switch noupdate

File: openupgrade_scripts/scripts/digest/17.0.1.1/pre-migration.py (hunk 1 above)
Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/scripts/digest/17.0.1.1/pre-migration.py:13]: Yeah

**Comment by hbrunn** [comment]
superseded by #4642

**Pull request closed without merging**